server/requirements.txt

```diff
@@ -0,0 +1,5 @@
+fastapi
```

Comment:
We have server_requirements.txt file, why we have another file with requirements?